### ISSUE TITLE
Pass data from menu to topview

### ISSUE THREAD

**Issue by rockia** [ECSlidingViewController/ECSlidingViewController#99]
Hello, thanks for the great Open Source library!! I have encountered a problem and hopefully you can give me some hints. (Sorry this is not actually a bug/issue about this git. Please remove if not appropriate).

I have the following 3 UIView objects, each associated a view object built with Storyboard
1. initView is the ECSlidingViewController
2. leftMenu is the leftView (UITableView)
3. centerMenu is the topview (UITableView)

First the initView loads leftView and topView, I can pan to see leftMenu, if I select a row on the leftMenu, can I pass data to the centerMenu.

Basically it's like a two levels navigation. I googled lots of tutorials but most of them are just initiate a new UIView after selecting from the menu. Can you please give me a hand? Thanks!
